system-addon/test/unit/content-src/components/Search.test.js

```diff
@@ -0,0 +1,56 @@
+const React = require("react");
+const {shallow, mount} = require("enzyme");
+const {_unconnected: Search} = require("content-src/components/Search/Search");
+const {actionTypes: at, actionUtils: au} = require("common/Actions.jsm");
+
+const DEFAULT_PROPS = {
+  Search: {
+    currentEngine: {
+      name: "google",
+      icon: "google.jpg"
+    }
+  },
+  dispatch() {}
+};
+
+describe("<Search>", () => {
+  it("should render a Search element", () => {
+    const el = shallow(<Search {...DEFAULT_PROPS} />);
+    assert.ok(el);
```

Comment:
Similar to the other comment, this might want to be something like wrapper.hasClass() or some such for more specificity and better readability.